api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/LocationTranslatorImpl.java

```diff
@@ -19,16 +19,40 @@
 @Setter(AccessLevel.PACKAGE)
 public class LocationTranslatorImpl implements LocationTranslator {
 	
+	private LocationAddressTranslatorImpl translator;
```

Comment:
This should be:

```java
@Inject
private LocationAddressTranslator translator;
```

This allows implementations to swap in their own `LocationAddressTranslator` if they need to override the default behaviour.